cloudflare-waf/info.json

```diff
@@ -1054,6 +1054,180 @@
         },
         "success": ""
       }
+    },
+    {
+      "title": "Create IP Items List",
+      "operation": "create_ip_items_list",
+      "description": "Add or append new items, such as IP addresses, to a specific list",
+      "annotation": "create_ip_items_list",
+      "category": "investigation",
+      "parameters": [
+        {
+          "title": "List ID",
+          "type": "text",
+          "name": "list_id",
+          "description": "Specifies the ID of the list from which you want to retrieve the list of IP addresses.",
```

Comment:
Please update description + tooltip: this action adds or appends new items rather than retrieving